### PR TITLE
gitui 0.8.0

### PR DESCRIPTION
**Pull request by extrawurst**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

File: Formula/gitui.rb [hunk 1]
```diff
@@ -22,7 +23,7 @@ def install
 
   test do
     system "git", "clone", "https://github.com/extrawurst/gitui.git"
-    (testpath/"gitui").cd { system "git", "checkout", "v0.7.0" }
+    (testpath/"gitui").cd { system "git", "checkout", "v0.8.0" }
```
Comment (SMillerDev) [inline Formula/gitui.rb:26]: Please use
```suggestion
    (testpath/"gitui").cd { system "git", "checkout", "v#{version}" }
```

File: Formula/gitui.rb [hunk 2]
```diff
@@ -40,8 +41,8 @@ def install
     # remove ANSI colors
     screenlog.gsub! /\e\[([;\d]+)?m/, ""
     assert_match "Author: Stephan Dilly", screenlog
-    assert_match "Date: 2020-06-15", screenlog
-    assert_match "SHA: 9c2a31846c417d8775a346ceaf38e77b710d3aab", screenlog
+    assert_match "Date: 2020-07-06", screenlog
+    assert_match "SHA: 2c53c0948788adc732fe5273f0288065224cac4d", screenlog
```
Comment (SMillerDev) [inline Formula/gitui.rb:45]: You should either always check the same tag, or don't check the hash/date. This is really difficult to keep up to date.

File: Formula/gitui.rb (hunk 1 above)
Comment (extrawurst) [reply Formula/gitui.rb:26]: After consulting with @vladimyr  (the original author of the formulae) over at https://github.com/extrawurst/gitui/pull/137 I tend to follow his strategy and not update the version on each release unless a breaking change requires it.

File: Formula/gitui.rb (hunk 2 above)
Comment (extrawurst) [reply Formula/gitui.rb:45]: @SMillerDev  see above

**Comment by extrawurst** [comment]
@SMillerDev I think it should be good to merge now

**Comment by extrawurst** [comment]
closing in favour of https://github.com/Homebrew/homebrew-core/pull/57573

**Comment by vladimyr** [comment]
@extrawurst You were simply faster than me. I was going to propose that you incorporate my changes and we go another way around 🤷

**Pull request closed without merging**